src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import light from "./logo-light-theme.webp";
+import dark from "./logo-dark-theme-removebg-preview-_1_.webp";
+import styles from './header.module.css'
+import styleTog from './toggler.module.css';
+
+function Header({ handleTheme, theme, searchValue, setSearchValue }) {
+  return (
+    <>
+      <header>
+        <div role="img" aria-label="Logo" className={styles.logo}>
+          <img
+            id="logoHandle"
+            src={theme ? dark : light}
+            role="logo"
+            alt="todoistic logo"
+          />
+        </div>
+        <div className={styles.searchHandle}>
+          <label htmlFor="seach">
+            <i className="fa-solid fa-magnifying-glass" />
+            <input
+              type="text"
+              id="search"
+              className={styles.search}
+              value={searchValue}
+              onChange={(e) => setSearchValue(e.target.value)}
+              placeholder="Search Tasks"
+            />
+          </label>
+        </div>
+        <label className={styleTog.toggle}>
```

Comment:
what does styleTog mean?

Reply:
I change it to this, <label className={stylesToToggler.toggle}>
by hurry, I named css module by "styleTog"